Resources/Prototypes/DeltaV/Entities/Mobs/Species/harpy.yml

```diff
@@ -127,6 +128,12 @@
     templateId: digitigrade
   - type: HarpyVisuals
   - type: UltraVision
+  - type: Tag
```

Comment:
What does this do? Don't they inherit this from basemob?

Reply:
If I add ANY tags, I now have to copy the entire BaseHuman tag component. Because tags aren't inherited for stupid reasons. I'm adding the NoStepShoesRequired tag for Harpies because I removed their shoes slot. This tag is a little stupidly named, but all it does it make it so that they don't get stunned by walking over glass shards. I assume the birbs with their tiny light feet just tip-claw around the shards, unlike how a clumsy human might stomp right into them.